bigchaindb/backend/localmongodb/schema.py

```diff
@@ -143,6 +144,14 @@ def create_abci_chains_indexes(conn, dbname):
                                                   name='height',
                                                   unique=True,)
 
+
+def create_elections_secondary_index(conn, dbname):
+    logger.info('Create `elections` secondary index.')
+
+    conn.conn[dbname]['elections'].create_index('election_id',
+                                                name='election_id',
+                                                unique=True,)
+
     logger.info('Create `abci_chains.chain_id` secondary index.')
```

Comment:
These 2 lines belong to the function above.

Reply:
Done